lib/libsh/libsimbol/libsimbol.sh

```diff
@@ -351,6 +354,50 @@ function core:softimport() {
     return $e
 }
 
+function core:module_path() {
+    local -i e=${CODE_SUCCESS?}
+
+    local path
+    if [ $# -eq 1 ]; then
+        local module="$1"
+
+        if [ -e ${SIMBOL_CORE_MOD?}/${module//\./\/}.sh ]; then
```

Comment:
Did you consider a case statement? Maybe later on we want to put in a warning if a module with the same name exists in CORE and USER>